### ISSUE TITLE
ClassArgumentMetadataTransfer value accept only string

### ISSUE THREAD

**Issue by ynnoig** [spryker-sdk/integrator#51]
As you can see here [https://github.com/spryker-sdk/integrator/blob/c587523495ed7d0aa99e6f116db1fa4d838cc9fb/src/Transfer/ClassArgumentMetadataTransfer.php#L84](https://github.com/spryker-sdk/integrator/blob/c587523495ed7d0aa99e6f116db1fa4d838cc9fb/src/Transfer/ClassArgumentMetadataTransfer.php#L84)

setValue is configured as strict and accept only string|null.

But here [https://github.com/spryker-sdk/integrator-manifests/blob/fc007c5ce8d1a8f2647e7e10af83e57d36e608fc/Spryker/DocumentationGeneratorOpenApi/0.1.0/installer-manifest.json#L21](https://github.com/spryker-sdk/integrator-manifests/blob/fc007c5ce8d1a8f2647e7e10af83e57d36e608fc/Spryker/DocumentationGeneratorOpenApi/0.1.0/installer-manifest.json#L21) is the value defined as array.

This cause that run `vendor/bin/integrator` a fatal error as followed:

```
Fatal error: Uncaught TypeError: SprykerSdk\Integrator\Transfer\ClassArgumentMetadataTransfer::setValue(): Argument #1 ($value) must be of type ?string, array given, called in /project/vendor/spryker-sdk/integrator/src/Builder/ClassMetadataBuilder/ClassMetadataBuilder.php on line 112 and defined in /project/vendor/spryker-sdk/integrator/src/Transfer/ClassArgumentMetadataTransfer.php:84
Stack trace:
#0 /project/vendor/spryker-sdk/integrator/src/Builder/ClassMetadataBuilder/ClassMetadataBuilder.php(112): SprykerSdk\Integrator\Transfer\ClassArgumentMetadataTransfer->setValue(Array)
#1 /project/vendor/spryker-sdk/integrator/src/Builder/ClassMetadataBuilder/ClassMetadataBuilder.php(97): SprykerSdk\Integrator\Builder\ClassMetadataBuilder\ClassMetadataBuilder->createClassArgumentMetadataTransfer(Array)
#2 /project/vendor/spryker-sdk/integrator/src/Builder/ClassMetadataBuilder/ClassMetadataBuilder.php(31): SprykerSdk\Integrator\Builder\ClassMetadataBuilder\ClassMetadataBuilder->addAppendArguments(Array, Object(SprykerSdk\Integrator\Transfer\ClassMetadataTransfer))
#3 /project/vendor/spryker-sdk/integrator/src/ManifestStrategy/WirePluginManifestStrategy.php(84): SprykerSdk\Integrator\Builder\ClassMetadataBuilder\ClassMetadataBuilder->build(Array)
#4 /project/vendor/spryker-sdk/integrator/src/Executor/ManifestExecutor.php(93): SprykerSdk\Integrator\ManifestStrategy\WirePluginManifestStrategy->apply(Array, 'Spryker.Documen...', Object(SprykerSdk\Integrator\Dependency\Console\SymfonyConsoleInputOutputAdapter), false)
#5 /project/vendor/spryker-sdk/integrator/src/IntegratorFacade.php(29): SprykerSdk\Integrator\Executor\ManifestExecutor->runModuleManifestExecution(Array, Object(SprykerSdk\Integrator\Dependency\Console\SymfonyConsoleInputOutputAdapter), false)
#6 /project/vendor/spryker-sdk/integrator/src/Console/ModuleInstallerConsole.php(67): SprykerSdk\Integrator\IntegratorFacade->runInstallation(Array, Object(SprykerSdk\Integrator\Dependency\Console\SymfonyConsoleInputOutputAdapter), false)
#7 /project/vendor/symfony/console/Command/Command.php(298): SprykerSdk\Integrator\Console\ModuleInstallerConsole->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /project/vendor/symfony/console/Application.php(1028): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 /project/vendor/symfony/console/Application.php(299): Symfony\Component\Console\Application->doRunCommand(Object(SprykerSdk\Integrator\Console\ModuleInstallerConsole), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /project/vendor/symfony/console/Application.php(171): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /project/vendor/spryker-sdk/integrator/bin/integrator(30): Symfony\Component\Console\Application->run()
#12 /project/vendor/bin/integrator(120): include('/project/vendor...')
#13 {main}
  thrown in /project/vendor/spryker-sdk/integrator/src/Transfer/ClassArgumentMetadataTransfer.php on line 84
```

**Comment by dereuromark** [comment]
Tracked as https://spryker.atlassian.net/browse/PPLUS-1767

**Comment by DmytroKlymanSpryker** [comment]
The issue is resolved and available in SDK version 0.6.9 or higher.
